user/manager.go

```diff
@@ -16,4 +16,5 @@ type Manager interface {
 	SetPrincipalOnCurrentUserByUserID(userID string, principal v3.Principal) (*v3.User, error)
 	CreateNewUserClusterRoleBinding(userName string, userUID apitypes.UID) error
 	GetUserByPrincipalID(principalName string) (*v3.User, error)
+	GetToken(clusterName, tokenName, description, kind, userName string) (*v3.Token, error)
```

Comment:
Does this method return all kinds of Token? Or is it specifically saml token or kubeconfig type?

Reply:
It returns based on the input `kind`. Right now it's kubeconfig but I didn't include that in func name to keep it generic